builtin/mainmenu/dlg_config_world.lua

```diff
@@ -33,14 +33,25 @@ local function get_formspec(data)
 
 	local hard_deps, soft_deps = pkgmgr.get_dependencies(mod.path)
 
+	if mod.is_modpack then
+		local info = pkgmgr.get_modpack_description(mod.path)
+		if info == "" then
+			info = fgettext("No modpack description provided.")
+		end
+		retval = retval ..
+			"label[0,0.7;" .. info .. "]"
```

Comment:
Needs wrapping/scrollability - see the content tab for an implementation. Make sure to remove any backgrounds for it though

Reply:
fixed